Resources/doc/reference/installation.rst

```diff
@@ -153,17 +153,18 @@ use it yet.
 
 If, at this point or during the installation, you come across any errors, don't panic:
 
-    -  Read the error message carefully. Try to find out exactly which bundle is causing the error. Is it SonataAdminBundle or one of the dependencies?
-    - Make sure you followed all the instructions correctly, for both SonataAdminBundle and its dependencies
+    - Read the error message carefully. Try to find out exactly which bundle is causing the error. Is it SonataAdminBundle or one of the dependencies?
+    - Make sure you followed all the instructions correctly, for both SonataAdminBundle and its dependencies.
     - Odds are that someone already had the same problem, and it's documented somewhere. Check Google_, `Sonata Users Group`_, `Symfony2 Users Group`_ and `Symfony Forum`_ to see if you can find a solution.
```

Comment:
Symfony2 Users Group and Symfony Forum are both dead. These are the only official and active support options: http://symfony.com/support